Update blocked events faq to include api calls (#5456)
segmentio/segment-docs: src/protocols/faq.md

### Do blocked and discarded events count towards my MTU counts?

> MTUs and API calls are calculated from the same pipeline. Blocking events within the source schema or tracking plan functionality (Protocols) will exclude it from API call calculations and MTU calculations as the events are discarded before they reach our pipeline that we use for calculations..

Reviewer: ```suggestion
Blocking events within the [Source Schema](/docs/connections/sources/schema/) or [Tracking Plans](/docs/protocols/tracking-plan/create/) excludes them from API call and MTU calculations, as the events are discarded before they reach the pipeline that Segment uses for calculations.
```
